addons/web/static/src/views/form/button_box/button_box.js

```diff
@@ -27,8 +30,13 @@ export class ButtonBox extends Component {
                 this.isFull = allVisibleButtons.length === maxVisibleButtons;
             } else {
                 // -1 for "More" dropdown
-                this.visibleButtons = allVisibleButtons.slice(0, maxVisibleButtons - 1);
-                this.additionalButtons = allVisibleButtons.slice(maxVisibleButtons - 1);
+                this.visibleButtons = allVisibleButtons.slice(
+                    0,
+                    Math.max(maxVisibleButtons - 1, 0)
+                );
+                this.additionalButtons = allVisibleButtons.slice(
+                    Math.max(maxVisibleButtons - 1, 0)
+                );
```

Comment:
Is not better to add a variable ?
```suggestion
                const splitIndex =  Math.max(maxVisibleButtons - 1, 0);
                this.visibleButtons = allVisibleButtons.slice(0, splitIndex);
                this.additionalButtons = allVisibleButtons.slice(splitIndex);
```